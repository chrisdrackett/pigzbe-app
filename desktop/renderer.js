import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
