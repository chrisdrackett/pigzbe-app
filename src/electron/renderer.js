import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
