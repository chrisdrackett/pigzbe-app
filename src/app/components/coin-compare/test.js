import React from 'react';
import renderer from 'react-test-renderer';
import Coin from './';

const props = {
    coin: 'BTC',
    value: '1231.12312'
};

describe('Coin', () => {
    test('renders correctly', () => {
        renderer.create(<Coin {...props}/>);
    });
});
