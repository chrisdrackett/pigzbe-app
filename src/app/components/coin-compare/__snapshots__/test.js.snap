// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coin renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexShrink": 0,
      "justifyContent": "center",
      "marginBottom": 20,
      "minWidth": "26%",
      "width": "26%",
    }
  }
>
  <Image
    style={
      Array [
        Object {
          "height": 35,
          "width": 35,
        },
        Object {
          "marginBottom": 15,
        },
      ]
    }
  />
  <Text
    style={
      Object {
        "color": "rgb(74, 74, 74)",
        "fontFamily": "Poppins",
        "fontSize": 14,
        "fontWeight": "bold",
      }
    }
  >
    btc
  </Text>
  <Text
    style={
      Object {
        "color": "rgba(74, 74, 74, 0.5)",
        "fontFamily": "Poppins",
        "fontSize": 14,
        "fontWeight": "bold",
      }
    }
  >
    1,231.12
  </Text>
</View>
`;
